ci/jenkins/jobs/macros.yaml

```diff
@@ -293,3 +293,12 @@
           set -e
           DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
           ./ci/jenkins/test.sh --testcase '{conformance_type}' --registry ${{DOCKER_REGISTRY}} --testbed-type jumper
+
+- builder:
+     name: builder-stop-all
+     builders:
+       - shell: |-
+           #!/bin/bash
+           set -e
+           chmod +x ./ci/jenkins/stop-stale-jobs.sh
```

Comment:
I don't think you need this if you change the file mode via `chmod +x ./ci/jenkins/stop-stale-jobs.sh` locally and push the file mode change via git.

Reply:
Not addressed?

Reply:
done